.charts/overwhelm/templates/overwhelm-system-namespace.yaml

```diff
@@ -1,6 +1,9 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    control-plane: controller-manager
-  name: overwhelm-system
+{{- if not .Values.overwhelm.namespace.create }}
+{{- else }}
```

Comment:
```suggestion
{{- if .Values.overwhelm.namespace.create }}
```

Reply:
I suppose this is the same thing, for some reason I thought I was being smart haha